### PR TITLE
Move to travis-ci.com

### PR DESCRIPTION
**Pull request by ennru**
References https://github.com/akka/akka-meta/issues/143

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.org/akka/akka.svg?branch=master)](https://travis-ci.org/akka/akka) 
+Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.com/akka/akka.svg?branch=master)](https://travis-ci.com/akka/akka)
```
Comment (raboof) [inline README.md:1]:
```suggestion
Akka [![Latest version](https://index.scala-lang.org/akka/akka/akka-actor/latest.svg)](https://index.scala-lang.org/akka/akka/akka-actor)[![Build Status](https://travis-ci.com/akka/akka.svg?branch=master)](https://travis-ci.com/github/akka/akka)
```

**Review by raboof — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ignasi35) [reply README.md:1]: :-o is that `github/` new?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -574,7 +574,7 @@ Akka currently uses a combination of Jenkins and Travis for Continuous Integrati
 
 * Jenkins [runs the tests for each PR](https://jenkins.akka.io:8498/job/pr-validator-per-commit-jenkins/)
 * Jenkins [runs a nightly test suite](https://jenkins.akka.io:8498/view/Nightly%20Jobs/job/akka-nightly/)
-* Travis [checks dependency licenses for all PR's](https://travis-ci.org/akka/akka)
+* Travis [checks dependency licenses for all PR's](https://travis-ci.com/akka/akka)
```
Comment (ignasi35) [inline CONTRIBUTING.md:577]: This probably needs a `github/` too.

**Review by ignasi35 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (raboof) [reply README.md:1]: not very new, but the old url redirects (though breaking the back button IIRC)

File: CONTRIBUTING.md (hunk 2 above)
Comment (raboof) [inline CONTRIBUTING.md:577]:
```suggestion
* Travis [checks dependency licenses for all PR's](https://travis-ci.com/github/akka/akka)
```

**Pull request merged**